### DISCUSSION TITLE
Installer Chamilo sur un serveur local mais accessible depuis internet

### DISCUSSION THREAD

**Discussion by YahnLP** [chamilo/chamilo-lms#5537] [General]
Bonjour à tous, 
Tout est déjà dans le titre. J'essaye d'Installer Chamilo sur un serveur local mais accessible depuis internet. J'ai bien réussi à le rendre accessible depuis internet MAIS quand j'essaye de créer un cours j'ai un message d'erreur me disant que la page n'existe pas.
Je pense que j'ai loupé quelque chose. J'ai lu la documentation d'installation mais je ne trouve pas de piste. Quelqu'un a-t-il déjà effectuer une telle installation ? 
Pour info, mon serveur est en linux Ubuntu dernière version avec Apache et MariaDB. La version de chamilo installée est la dernière en date. Je passe par une freebox avec ports 80 et 443 redirigés vers ma machine.
Si quelqu'un à un mode opératoire je suis preneur pour voir où je me suis loupé.

**Comment by Moigaston** [comment]
Salut!!

> quand j'essaye de créer un cours j'ai un message d'erreur me disant que la page n'existe  pas.

>

Ce serait utile de poster l'erreur exacte qui est renvoyée! 

Est-ce celle-ci:  

> 404 Not Found The requested URL was not found on this server

**Comment by Coursenligne** [comment]
9 chances sur 10 que Chamilo répond sur une  URL mais qu'après, tout soit considéré comme local (local host ou 127.0.0.1) ce qui correspond à l'adresse de bouclage et donc ne répond pas sur Internet...
A vous lire (sans plus d'information) c'est la première chose à laquelle je pense.

**Comment by Coursenligne** [comment]
Bon je pense que le souci est résolu, mais c'est dommage de n'avoir pas eu de retour sur l'erreur faite, ça peut toujours aider les autres....

**Comment by YahnLP** [comment]
En fait non,  toujours pas résolu. J'analyse les fichiers. Mais j'ai des
doutes sur la faisabilité.

Le ven. 16 août 2024, 17:56, Coursenligne ***@***.***> a
écrit :

**Comment by Coursenligne** [comment]
C'est quoi l'environnement du serveur local ?
Ne serai-ce pas l'URL mal renseignée dans le fichier de configuration ?
